dev_tools/scripts/db_test.sh

```diff
@@ -4,6 +4,10 @@ docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submissio
 
 CONFIGFILE="dev_tools/config.yaml" go run . --action pg_dump
 
+if [ $? != 0 ]; then
+    exit 1
```

Comment:
Might be nice for extra clarity to make clear why we bail out (hopefully evident from the failing tool, but does certainly not hurt to be clear about bailing out because of pre-requisites that fail).